# -*- coding: utf-8 -*-
"""Vaccinations_country.ipynb

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1q3elpTQez4887Z76vUJFeWtVKzBida2C
"""

import json
import urllib.request
from urllib.error import HTTPError
import pandas as pd 
def main():
  url_add = 'https://storage.googleapis.com/covid19-open-data/v2/vaccinations.json'
  try:

    with urllib.request.urlopen(url_add) as url:
      data = json.loads(url.read())

  except HTTPError as ex:
    print(ex.read())
  df = pd.DataFrame(data['data'], columns = data['columns'])
  df = df.loc[~df['key'].str.contains('_')]
  return df.to_json('vaccinations_country.json', orient = 'split', compression = 'infer', index = 'true')

main()